python-lib/tc_etl_lib/tc_etl_lib/auth.py

```diff
@@ -61,60 +63,67 @@ def __init__(self,*, endpoint: str = None, service: str = None, user: str = None
 
         if (password == None):
             messageError.append('<<password>>')
-        
+
         if len(messageError) != 0:
             defineParams = messageError[0]
             if len(messageError) != 1:
                 defineParams = " and ".join([", ".join(messageError[:-1]), messageError[-1]])
             raise ValueError(f'You must define {defineParams} in authManager')
-        
+
         self.endpoint = endpoint
         self.service = service
         self.user = user
         self.password = password
         self.subservice = subservice
-        self.tokens = {}  
+        self.tokens = {}
+
+    def get_token(self, subservice):
+        if not self.defaultToken is None:
+            return self.defaultToken
+        return self.tokens[subservice]
+
+    def set_token(self, token: str):
```

Comment:
```suggestion
    def set_token(self, subservice: str, token: str):
```

La función añadiría o reemplazaría dentro del objeto `tokens` el token suministrado como argumento para el subservicio suministrado como argumento.